spark/src/main/java/org/apache/iceberg/spark/source/IcebergSource.java

```diff
@@ -149,6 +149,12 @@ protected Table findTable(DataSourceOptions options, Configuration conf) {
     }
   }
 
+  private boolean routeToHadoopTable(final String path) {
```

Comment:
Can you give examples of paths that you think are handled incorrectly by the current logic?

Reply:
I thought path = "db.tab/le" (table name or db name contains "/") might be better to be routed to Hive table instead of Hadoop table (as the current logic). But as I tried, Hive do not allow us to create a db or a table with "/" in its name. This PR seems not necessary.